composer.json

```diff
@@ -15,12 +15,17 @@
         }
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.5.0"
```

Comment:
This is not good. This change break BC and require major release.

Reply:
Agree. Revert this.